third_party/stdlib/collections.py

```diff
@@ -258,37 +258,43 @@ def viewitems(self):
 ### namedtuple
 ################################################################################
 
-_class_template = '''\
-class {typename}(tuple):
+class NamedTuple(tuple):
     '{typename}({arg_list})'
 
     __slots__ = ()
 
-    _fields = {field_names!r}
+    _fields = ()
 
-    def __new__(_cls, {arg_list}):
+    def __new__(_cls, *args, **kwds):
         'Create new instance of {typename}({arg_list})'
-        return _tuple.__new__(_cls, ({arg_list}))
+        try:
+            kwargs = tuple(map(kwds.pop, list(_cls._fields[len(args):])))
```

Comment:
Sure. Instead of:

        try:
        	kwargs = tuple(map(kwds.pop, list(_cls._fields[len(args):])))
        except KeyError as e:
            raise TypeError('Unexpected keyword argument %r' % e.message)
        return tuple.__new__(_cls, args + kwd_values)

More clear?

        try:
        	keywords = list(_cls.fields[len(args):])
        	kwd_values = tuple(map(kwds.pop, keywords))
        except KeyError as e:
            raise TypeError('Unexpected keyword argument %r' % e.message)
        return tuple.__new__(_cls, args + kwd_values)